activiti-core/activiti-bpmn-converter/src/main/java/org/activiti/bpmn/converter/child/LinkEventTargetParser.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2010-2020 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.activiti.bpmn.converter.child;
+
+import org.activiti.bpmn.model.BaseElement;
+import org.activiti.bpmn.model.BpmnModel;
+import org.activiti.bpmn.model.LinkEventDefinition;
+
+import javax.xml.stream.XMLStreamReader;
+
+public class LinkEventTargetParser extends BaseChildElementParser {
+
+    public String getElementName() {
+        return ATTRIBUTE_LINK_TARGET;
+    }
+
+    public void parseChildElement(XMLStreamReader xtr, BaseElement parentElement, BpmnModel model) throws Exception {
+        if (!(parentElement instanceof LinkEventDefinition)) {
+            return;
+        }
+
+        LinkEventDefinition eventDefinition = (LinkEventDefinition) parentElement;
+        eventDefinition.setTarget(xtr.getElementText());
+    }
```

Comment:
It can be simplified using pattern matching

Reply:
Done